www/FileZilla/Portfile

```diff
@@ -25,13 +30,41 @@ checksums           rmd160  950e44a39b4464cd47888295eb20cb394b8df6fd \
                     sha256  a40f04e02efaae7b50d1515ee1c36c4b0e445818566c450e440bfd6c70e9b203 \
                     size    4057952
 
-# wxWidgets-3.0 with support for C++11 on < 10.9
-wxWidgets.use       wxWidgets-3.0-cxx11
+# wxWidgets-3.0 with support for C++11 on < 10.9.
+# However, on < 10.8 wxWidgets-3.0* do not work correctly.
+# So for older macOS (and non-Darwin) use wxGTK.
+if {${os.platform} eq "darwin" && ${os.major} > 11} {
+    wxWidgets.use   wxWidgets-3.0-cxx11
+} else {
+    wxWidgets.use   wxGTK-3.0-cxx11
+
+    depends_lib-append \
+                    port:desktop-file-utils
+
+    legacysupport.redirect_bins filezilla
+
+    # Contrary to documentation, FZ_DATADIR does nothing
+    # on macOS, since assumption of app bundle is hardcoded.
+    patchfiles-append \
+                    patch-fix-fz_datadir.diff
+}
+
+if {[string match *gcc* ${configure.compiler}]} {
+    # Yes, this is gcc-specific, or at least configure thinks so.
+    # configure: error: Boost Regex 1.76 or higher not found.
+    PortGroup       boost 1.0
+
+    boost.version   1.81
+}
```

Comment:
this is weird... is boost needed or not? That should not be specific to whether gcc or clang is used. It seems to me this needs a bit more work to sort out properly.

Reply:
The build system wants to use boost-regex if GCC is used. Configure fails, not being able to find Boost (without PG it is not found). Minimum required is 1.76, but I see no reason not to use a newer one.

Reply:
but does it actually link to it, and, if so, doesn't that happen when using Clang (perhaps the it is just found correctly). In other words, you build it with Clang in trace-mode? It seems very unlikely that this is a GCC only issue..

Reply:
@reneeotten Perhaps it is a build-time dependency, and you are right to raise this issue. It should be set to build_dep only. I will sort this now.

Reply:
@reneeotten A little delay, I am trying to update it to the current version now. If that works, I will make an update here. Boost will be taken care of anyway.

Reply:
Fixed